### ISSUE TITLE
Audio range too narrow

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#55]
```
What steps will reproduce the problem?
1. Turn sound to 1 notch in OS X
2. Adjust input to an appropriate level for 1 notch
3. Turn sound in OS X to maximum level
4. Observe that the audio is not loud enough for maximum level

What is the expected output? What do you see instead?
There should be a much greater difference between 1 notch in OS X and full.

What version of the product are you using? On what operating system?
1.5.1

Please provide any additional information below.
Basically the low end of the sound level needs to be quieter and the high end 
needs to be louder, 
so that I can stop fiddling with the sound level on my usb audio interface 
every time I want to 
listen to my music really loud or really quiet.

```

Original issue reported on code.google.com by `troydona...@gmail.com` on 23 Dec 2009 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
For the next version the dynamic range has been increased from 18 dB to 72 dB.
```

Original comment by `74obje...@gmail.com` on 9 Mar 2010 at 4:11
- Changed state: **Fixed**
